app/models/elasticsearch_service.rb

```diff
@@ -69,7 +70,7 @@ def search_facilities(params)
     validate_search(params)
 
     size = params[:size].to_i
-    size = 50 if size == 0
+    size = 1000 if size == 0
```

Comment:
With this page size are we even stressing all the pagination flows? Wouldn't 500 work?